.github/workflows/database-backup.yml

```diff
@@ -115,3 +115,68 @@ jobs:
           name: backup_sanitised
           path: backup_sanitised.sql
           retention-days: 3
+
+    job2:
+      name: Restore backup to Preprod
+      runs-on: ubuntu-latest
+      needs: [job1]
+      services:
+      postgres:
+        image: postgres:13.9
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+      environment:
+      name: preprod
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - uses: Azure/login@v1
+        with:
+          creds: ${{ secrets.azure_credentials }}
+
+      - name: Set environment variables Production
+        shell: bash
+        run: |
+          tf_vars_file=terraform/workspace_variables/preprod.tfvars.json
+          echo "KEY_VAULT_NAME=$(jq -r '.key_vault_name' ${tf_vars_file})" >> $GITHUB_ENV
+          echo "PAAS_SPACE=$(jq -r '.paas_space' ${tf_vars_file})" >> $GITHUB_ENV
+
+      - uses: Azure/get-keyvault-secrets@v1
+        id: get_secrets
+        with:
+          keyvault: ${{ env.KEY_VAULT_NAME }}
+          secrets: "BACKUP-STORAGE-CONNECTION-STRING,PAAS-USER,PAAS-PASSWORD"
+
+      - uses: DfE-Digital/keyvault-yaml-secret@v1
```

Comment:
What is the `SLACK_WEBHOOK` used for in this job?

Reply:
we dont need it

Reply:
removed